triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -50,10 +50,9 @@ import com.daml.logging.{ContextualizedLogger, LoggingContextOf}
 import LoggingContextOf.{label, newLoggingContext}
 import com.daml.platform.participant.util.LfEngineToApi.toApiIdentifier
 import com.daml.platform.services.time.TimeProviderType
-import com.daml.script.converter.Converter.{DamlTuple2, DamlAnyModuleRecord, unrollFree}
+import com.daml.script.converter.Converter.{DamlAnyModuleRecord, DamlTuple2, unrollFree}
 import com.daml.script.converter.Converter.Implicits._
 import com.daml.script.converter.ConverterException
-
 import com.google.protobuf.empty.Empty
```

Comment:
small nit, that whitespace change is not necessary

Reply:
Computer did it, not me. I defer to IntelliJ for things like this.

Reply:
no worries here, this constantly happens to me too with IntelliJ and I expected that this was IntelliJ too as it is in the classic `import` statements area :smile:

Reply:
@SamirTalwar-DA IntelliJ's decisions here are not idempotent, so this will never eventually settle on an "IntelliJ-clean" state. Best to leave them out entirely with `git add -p`.